tendrl/commons/utils/ansible_module_runner.py

```diff
@@ -111,9 +111,8 @@ def run(self):
             result = json.loads(out)
 
         except (subprocess.CalledProcessError, ValueError) as ex:
-            result = repr(ex)
-            err = result
+            result = {'stderr': repr(ex)}
```

Comment:
How does it take care of `result.get('stdout')` in calling code now?

Reply:
Since, a dictionary is being returned now, the stdout will be assigned blank string( result.get("stdout", "") ), thus the code will not throw any error. https://github.com/Tendrl/commons/blob/36458f15dcd7d06257f244aba84a3dbeb6d13ccb/tendrl/commons/utils/cmd_utils.py#L78